spec/examples.txt

```diff
@@ -1,5 +1,7 @@
 example_id                      | status | run_time        |
 ------------------------------- | ------ | --------------- |
-./spec/unit/item_spec.rb[1:1:1] | passed | 0.01463 seconds |
-./spec/unit/item_spec.rb[1:1:2] | passed | 0.00033 seconds |
-./spec/unit/item_spec.rb[1:1:3] | passed | 0.00037 seconds |
+./spec/unit/game_spec.rb[1:1:1] | passed | 0.01836 seconds |
```

Comment:
You should add spec/examples.txt into .gitignore. I forgot to do it in the first place.